src/test/java/io/nats/client/impl/ConnectionListenerTests.java

```diff
@@ -94,40 +85,32 @@ public void testDisconnectReconnectCount() throws Exception {
             int port;
             try (NatsTestServer ts = new NatsTestServer(false)) {
                 Options options = new Options.Builder().
-                                    server(ts.getURI()).
-                                    reconnectWait(Duration.ofMillis(100)).
-                                    maxReconnects(-1).
-                                    connectionListener(handler).
-                                    build();
+                        server(ts.getURI()).
+                        reconnectWait(Duration.ofMillis(100)).
+                        maxReconnects(-1).
+                        connectionListener(handler).
+                        build();
                 port = ts.getPort();
                 nc = Nats.connect(options);
-                assertTrue(Connection.Status.CONNECTED == nc.getStatus(), "Connected Status");
+                assertSame(Connection.Status.CONNECTED, nc.getStatus(), "Connected Status");
```

Comment:
assertSame is recommended for enum, even though it does almost the exact same thing. Intellij called it "simplify"